app/views/administrate/application/_collection.html.erb

```diff
@@ -29,10 +29,7 @@ to display a collection of resources in an HTML table.
         <%= link_to(params.merge(
           collection_presenter.order_params_for(attr_name)
         )) do %>
-        <%= t(
-          "helpers.label.#{resource_name}.#{attr_name}",
-          default: attr_name.to_s,
-        ).titleize %>
+            <%= field_label_text(resource_name, attr_name).titleize %>
```

Comment:
I think you should move `titleize` into `field_label_text`, because it can broke text retrieved from `human_attribute_name`